### PR TITLE
Profile specific implementation guidance: add missing, remove erronous, align with Base

### PR DESCRIPTION
**Pull request by dt-r**
- updated the guidance around use of Practitioner & PractitionerRole, also added to CapabilityStatements, removed reference to CareTeam
- added std identifiers guidance where it was missing
- updated sex and gender guidance in line with AU Base
- linked to Observation Grouping guidance in FHIR 
- some punctuation fixes

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 
 
 ### Profile specific implementation guidance
-- See the [Medicine Information](medicine-information.html) page for guidance
+- See the [Medicine Information](medicine-information.html) page for guidance.
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md:21]: This comment is related to the information guidance in Medications profile. Add full stop to the end of the information guidance that references the same page.

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md [hunk 2]
```diff
@@ -17,5 +17,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 
 
 ### Profile specific implementation guidance
-- See guidance on the construction of an identifier on the relevant Identifier profile page and the section on [Business Identifiers](https://build.fhir.org/ig/hl7au/au-fhir-base/guidance.html#business-identifiers) in AU Base.
+- Even though AU Core Profiles allow both PractitionerRole and Practitioner to be referenced, because PractitionerRole supplies a provider’s location, contact information, and reference to the Practitioner, it **SHOULD** be referenced instead of the Practitioner resource. 
+- If a system only supports Practitioner, the system **SHALL** provide implementation specific guidance how to access a provider’s location and contact information using only the Practitioner resource.
 - Requesters can request responders return the Practitioner resource using `_include`, see the Notes section below.
+- See guidance on the construction of an identifier on the relevant Identifier profile page and the section on [Business Identifiers](https://build.fhir.org/ig/hl7au/au-fhir-base/guidance.html#business-identifiers) in AU Base.
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md:23]: incorrect link, should link to https://build.fhir.org/ig/hl7au/au-fhir-base/generalguidance.html#business-identifiers

File: input/intro-notes/StructureDefinition-au-core-practitioner-intro.md [hunk 3]
```diff
@@ -17,5 +17,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 
 
 ### Profile specific implementation guidance
-- Because PractitionerRole supplies a provider’s location, contact information, and reference to the Practitioner, it **SHOULD** be referenced in `CareTeam.member` instead of the Practitioner resource. If a system only supports Practitioner, the system **SHALL** provide implementation specific guidance how to access a provider’s location and contact information using only the Practitioner resource.
+- Even though AU Core Profiles allow both PractitionerRole and Practitioner to be referenced, because PractitionerRole supplies a provider’s location, contact information, and reference to the Practitioner, it **SHOULD** be referenced instead of the Practitioner resource. 
+- If a system only supports Practitioner, the system **SHALL** provide implementation specific guidance how to access a provider’s location and contact information using only the Practitioner resource.
 - See guidance on the construction of an identifier on the relevant Identifier profile page and the section on [Business Identifiers](https://build.fhir.org/ig/hl7au/au-fhir-base/guidance.html#business-identifiers) in AU Base.
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-practitioner-intro.md:22]: incorrect link, should link to https://build.fhir.org/ig/hl7au/au-fhir-base/generalguidance.html#business-identifiers

**Review by dbojicic — CHANGES_REQUESTED** [review]
A few changes requested, please see inline comments.

File: input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md (hunk 1 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md:21]: Change made to AU Core Medication.

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md (hunk 2 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-intro.md:23]: Fixed all links to Business Identifiers section in AU Base

**Review by dbojicic — APPROVED** [review]

**Pull request merged**